pkg/common/jwtsvid/common.go

```diff
@@ -0,0 +1,23 @@
+package jwtsvid
+
+import (
+	"time"
+
+	jwt "github.com/dgrijalva/jwt-go"
+)
+
+func audienceClaim(audience []string) interface{} {
```

Comment:
the audience claims is one or more string values. in the common case, i.e. a single audience, the RFC allows for the value to be encoded as a single string instead of a list. this function returns either a list of strings, or, if the length is one, returns just the first element, hence an `interface{}` return value.